core/controllers/email_dashboard_test.py

```diff
@@ -152,6 +184,268 @@ def setUp(self):
         self.set_admins(
             [self.SUBMITTER_USERNAME, self.NEW_SUBMITTER_USERNAME])
 
+    def test_handler_with_invalid_num_queries_to_fetch_raises_error_400(self):
+        self.login(self.SUBMITTER_EMAIL)
+
+        response = self.get_json(
+            '/emaildashboarddatahandler',
+            params={'num_queries_to_fetch': 'invalid_num_queries_to_fetch'},
```

Comment:
Please use some kind of unreasonable number (maybe `NaN` or `-5`) to match the expected type.

Reply:
Done!